README.md

```diff
@@ -1,4 +1,4 @@
-# contxt-sdk [![wercker status](https://app.wercker.com/status/869ef086297da79ddd0cbf3564f7cba6/s/master 'wercker status')](https://app.wercker.com/project/byKey/869ef086297da79ddd0cbf3564f7cba6)
+# contxt-sdk
```

Comment:
```suggestion
# contxt-sdk ![Test](https://github.com/ndustrialio/contxt-sdk-js/workflows/Test/badge.svg)
```

Reply:
Thanks. I was trying to find the badges but I couldn't

Reply:
Took me a while to find it too: it's [here](https://github.com/ndustrialio/contxt-sdk-js/actions?query=workflow%3ATest) (top right corner, `Create status badge`)